### DISCUSSION TITLE
SVG support?

### DISCUSSION THREAD

**Discussion by lavrentius-dev** [vers-one/EpubReader#108] [Ideas]
This app is beautiful! Some stuff needs additional work (e.g., internal links don't seem to work). Any plans for SVG support? A NetMaui app?

Despite so many organizations being locked in the PDF era, EPUB is the future of document interchange formats: an open source, flow document format, with TOC and user-adjustable font face and size. 

Way to go.

**Comment by vers-one** [comment]
You mean the WPF demo app? The library itself [does support](https://os.vers.one/EpubReader/reference/VersOne.Epub.EpubContentType.html) SVG images.

The demo app was not meant to be a fully-fledged EPUB reader. Its main purpose was to demonstrate how to open EPUB files and access their content using the library.

[HtmlRenderer.WPF](https://github.com/ArthurHub/HTML-Renderer), the HTML rendering library used in the demo app, is quite limited (for example, it doesn't support HTML 5 and CSS 3 which are very common in the EPUB books nowadays) and is no longer maintained. It's possible to replace HtmlRenderer.WPF with something like Chromium Embedded Framework which would make the demo app more versatile, including the SVG rendering support, but that is currently outside the scope of EpubReader library development.

If you want to see an example of a production-quality EPUB reader web app that uses this library, I can recommend [Kavita](https://github.com/Kareadita/Kavita).
